Gopkg.lock

```diff
@@ -184,169 +233,220 @@
   version = "v1.0.0"
 
 [[projects]]
+  digest = "1:7231124c9669dfb54b82ef8b89f2735cf5d5d2529a23c6ac93a8c4b8bbb28b28"
   name = "github.com/pelletier/go-toml"
   packages = ["."]
+  pruneopts = "UT"
   revision = "acdc4509485b587f5e675510c4f2c63e90ff68a8"
   version = "v1.1.0"
 
 [[projects]]
   branch = "master"
+  digest = "1:4d92d3bcd412de705100c10f0428a0b63b12f3d12455ebae46e9ea384c23b333"
   name = "github.com/samuel/go-zookeeper"
   packages = ["zk"]
+  pruneopts = "UT"
   revision = "c4fab1ac1bec58281ad0667dc3f0907a9476ac47"
 
 [[projects]]
   branch = "master"
+  digest = "1:579c4bbcc2e16d4caf871ba91c0e2c331b07c5560c80d142d82c0de01c57fa96"
   name = "github.com/sean-/seed"
   packages = ["."]
+  pruneopts = "UT"
   revision = "e2103e2c35297fb7e17febb81e49b312087a2372"
 
 [[projects]]
+  digest = "1:fe0b7f0c9a5e5511001fe085b0a156b29266012cd984a63e4059a30e84bba03a"
   name = "github.com/spf13/afero"
   packages = [
     ".",
-    "mem"
+    "mem",
   ]
+  pruneopts = "UT"
   revision = "63644898a8da0bc22138abf860edaf5277b6102e"
   version = "v1.1.0"
 
 [[projects]]
+  digest = "1:516e71bed754268937f57d4ecb190e01958452336fa73dbac880894164e91c1f"
   name = "github.com/spf13/cast"
   packages = ["."]
+  pruneopts = "UT"
   revision = "8965335b8c7107321228e3e3702cab9832751bac"
   version = "v1.2.0"
 
 [[projects]]
   branch = "master"
+  digest = "1:080e5f630945ad754f4b920e60b4d3095ba0237ebf88dc462eb28002932e3805"
   name = "github.com/spf13/jwalterweatherman"
   packages = ["."]
+  pruneopts = "UT"
   revision = "7c0cea34c8ece3fbeb2b27ab9b59511d360fb394"
 
 [[projects]]
+  digest = "1:9424f440bba8f7508b69414634aef3b2b3a877e522d8a4624692412805407bb7"
   name = "github.com/spf13/pflag"
   packages = ["."]
+  pruneopts = "UT"
   revision = "583c0c0531f06d5278b7d917446061adc344b5cd"
   version = "v1.0.1"
 
 [[projects]]
+  digest = "1:59e7dceb53b4a1e57eb1eb0bf9951ff0c25912df7660004a789b62b4e8cdca3b"
   name = "github.com/spf13/viper"
   packages = ["."]
+  pruneopts = "UT"
   revision = "b5e8006cbee93ec955a89ab31e0e3ce3204f3736"
   version = "v1.0.2"
 
 [[projects]]
   branch = "master"
+  digest = "1:6014b4cd08abf98a4a6855e086837f21360362c2f59271aac3aad2b30b4f96f6"
   name = "github.com/streadway/amqp"
   packages = ["."]
+  pruneopts = "UT"
   revision = "8e4aba63da9fc5571e01c6a45dc809a58cbc5a68"
 
 [[projects]]
+  digest = "1:03aa6e485e528acb119fb32901cf99582c380225fc7d5a02758e08b180cb56c3"
   name = "github.com/ugorji/go"
   packages = ["codec"]
+  pruneopts = "UT"
   revision = "b4c50a2b199d93b13dc15e78929cfb23bfdf21ab"
   version = "v1.1.1"
 
 [[projects]]
+  digest = "1:076b7997c32e74635131eba607a543cc66bfce01805f6388ad704a2031975007"
   name = "github.com/victorcoder/dkron"
   packages = [
     "cron",
     "dkron",
-    "plugin"
+    "plugin",
+    "proto",
   ]
-  revision = "d2eb5174f85bdfe6b33094343651e8d4a6ea5f8b"
-  version = "v0.10.2"
+  pruneopts = "UT"
+  revision = "adf1c20ab86e5a3c03852581e505545d4e0e0d64"
+  version = "v0.10.4"
```

Comment:
this version should match the `dkron` version of the main image (which is `0.10.2`)

Reply:
I agree. I'm struggling with `dep`... I've juste ran a `dep ensure -v -update` to update the `Gopkg.lock` file and it update the `dkron` version even if there is a constraint in `Gopkg.toml`...

Reply:
I try to fix the version in `Gopkg.toml` using `  version = "=0.10.2"`. However the `dep ensure` fail to solve `dkron` dependency (because a weird error on submodules) : 

```
Solving failure: 
        (1) unexpected error while defensively cleaning up after possible derelict nested submodule directories: fatal: No url found for submodule path 'website/themes/docdock' in .gitmodules
: command failed: [git submodule foreach --recursive git clean -x -d -f -f]: exit status 128
        (2) unexpected error while defensively cleaning up after possible derelict nested submodule directories: fatal: No url found for submodule path 'website/themes/docdock' in .gitmodules
: command failed: [git submodule foreach --recursive git clean -x -d -f -f]: exit status 128
        (3) unexpected error while defensively cleaning up after possible derelict nested submodule directories: fatal: No url found for submodule path 'website/themes/docdock' in .gitmodules
: command failed: [git submodule foreach --recursive git clean -x -d -f -f]: exit status 128
        (4) unexpected error while defensively cleaning up after possible derelict nested submodule directories: fatal: No url found for submodule path 'website/themes/docdock' in .gitmodules
: command failed: [git submodule foreach --recursive git clean -x -d -f -f]: exit status 128
```

Besides we can't use the docker image `v0.10.4` because it does not exists.